docs/content/preview/secure/vulnerability-disclosure-policy.md

```diff
@@ -57,18 +57,57 @@ An important part of Yugabyte's strategy for building a secure platform for our
-| :------ | :--------- | :-- | :---------------- | :------- | :----- |
-| YugabyteDB | Fixed: update to {{<release "2.18.4.0">}} or later | {{<cve "CVE-2023-6001">}} | from v2.0 through {{<release "2.18.3.0">}} | {{<release "2.18.4.0">}} | Resolved |
-| YugabyteDB | Fixed: update to {{<release "2.14.14.0">}}, {{<release "2.16.8.0">}}, {{<release "2.18.4.0">}} and later | {{<cve "CVE-2023-6002">}} | from v2.0 through {{<release "2.14.13.0">}}, {{<release "2.16.7.0">}}, and {{<release "2.18.3.0">}} | {{<release "2.14.14.0">}}, {{<release "2.16.8.0">}}, {{<release "2.18.4.0">}} and later | Resolved |
-| YugabyteDB | Fixed: update to {{<release "2.17.4.0">}} or later | {{<cve "CVE-2023-4640">}} | from v2.0 through {{<release "2.17.3.0">}} | {{<release "2.17.4.0">}} | Resolved |
-| YugabyteDB | Fixed: update to {{<release "2.14.0.0">}} or later | {{<cve "CVE-2023-0745">}} | from v2.0 through [v2.13.0.0](/preview/releases/ybdb-releases/end-of-life/v2.13/) | {{<release "2.14.0.0">}} | Resolved |
-| YugabyteDB | Fixed: update to {{<release "2.15.0.0">}} or later | {{<cve "CVE-2023-0575">}} | from v2.0 through {{<release "2.14.0.0">}} | {{<release "2.15.0.0">}} | Resolved |
-| YugabyteDB | Fixed: update to {{<release "2.14.0.0">}} or later | {{<cve "CVE-2023-0574">}} | from v2.0 through [v2.13.0.0](/preview/releases/ybdb-releases/end-of-life/v2.13/) | {{<release "2.14.0.0">}} | Resolved |
-| YugabyteDB | Fixed: update to v2.6.1.1 or later | {{<cve "CVE-2022-37397">}} | v2.6.1.0 | v2.6.1.1 | Resolved |
+| Product | Name | Affected versions | Fixed in | Status |
+| :------ | :-- | :---------------- | :------- | :----- |
+| YugabyteDB | {{<cve "CVE-2023-6001">}} | from v2.0 through {{<release "2.18.3.0">}} | {{<release "2.18.4.0">}} | Resolved |
+| YugabyteDB | {{<cve "CVE-2023-6002">}} | from v2.0 through {{<release "2.14.13.0">}}, {{<release "2.16.7.0">}}, and {{<release "2.18.3.0">}} | {{<release "2.14.14.0">}}, {{<release "2.16.8.0">}}, {{<release "2.18.4.0">}} and later | Resolved |
+| YugabyteDB | {{<cve "CVE-2023-4640">}} | from v2.0 through {{<release "2.17.3.0">}} | {{<release "2.17.4.0">}} | Resolved |
+| YugabyteDB | {{<cve "CVE-2023-0745">}} | from v2.0 through [v2.13.0.0](/preview/releases/ybdb-releases/end-of-life/v2.13/) | {{<release "2.14.0.0">}} | Resolved |
+| YugabyteDB | {{<cve "CVE-2023-0575">}} | from v2.0 through {{<release "2.14.0.0">}} | {{<release "2.15.0.0">}} | Resolved |
+| YugabyteDB | {{<cve "CVE-2023-0574">}} | from v2.0 through [v2.13.0.0](/preview/releases/ybdb-releases/end-of-life/v2.13/) | {{<release "2.14.0.0">}} | Resolved |
+| YugabyteDB | {{<cve "CVE-2022-37397">}} | v2.6.1.0 | v2.6.1.1 | Resolved |
 
 {{< note title="Note" >}}
 
 Our [release notes](/preview/releases/ybdb-releases/) contain up-to-date information on security vulnerabilities and available patches.
 
 {{< /note >}}
+
+### Supply-chain vulnerabilities in YugabyteDB Postgres Query Layer (YSQL)
```

Comment:
I would add more details about the scope of this section. Example:

### Scope of Supply Chain Vulnerabilities

Please note that this policy covers only vulnerabilities in the query layer of PostgreSQL. We do not publish or disclose any other third-party (supply chain) vulnerabilities. Our focus is solely on ensuring the security and integrity of the PostgreSQL query layer.